<launch>
    <arg name = "gpsport"/>
    <node pkg = "gps_imu" name = "gps_sensor" output = "screen" args= "$(arg gpsport)" type = "gpsdriver.py"/>
    <arg name="serial_port"/> 
    <node name="imu" pkg="gps_imu" type="imudriver.py" output="screen" >
    <param name="port" value="$(arg port)"/>
    </node>
    
    <node name= "convert_to_quaternion" type="convert_to_quaternion.py" pkg= "gps_imu"/>
</launch>
